src/commands/workspace/list.ts

```diff
@@ -0,0 +1,24 @@
+import { flags as oclifFlags } from '@oclif/command'
+
+import { CustomCommand } from '../../oclif/CustomCommand'
+import workspaceList from '../../modules/workspace/list'
+
+export default class WorkspaceList extends CustomCommand {
+  static description = 'List workspaces on this account'
+
+  static aliases = ['workspace ls']
```

Comment:
```suggestion
  static aliases = ['workspace:ls']
```

Reply:
Then the alias will be showed with `:` in the doc, right? We will need to fix it.